app/models/miq_provision_request_template.rb

```diff
@@ -74,6 +74,7 @@ def get_parent_task(service_task)
   end
 
   def number_of_vms_from_dialog(root_svc, parent_task)
+    return nil unless root_svc.options[:dialog]
```

Comment:
We are intentionally returning `nil` here to say that there is no `number_of_vms` value defined in the dialog.  This prevents us from overwriting the value configured in the service_template.